CONTRIBUTING.md

```diff
@@ -108,7 +106,7 @@ Ready to contribute? Here's how to set up the project for local development.
 
     ```sh
     git add .
-    cz commit  # use `git commit` if you prefer, but this helps
+    poetry run cz commit  # use `git commit` if you prefer, but this helps
```

Comment:
Following your own line of reasoning in https://github.com/copier-org/copier/pull/767, this should be left as it was, isn't it?

Reply:
You're right, of course! :bow: I've reverted this change. Thanks! :+1: